molecule/default/tests/test_default.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
+
+
+def test_hosts_file(host):
```

Comment:
Missed this in my first review, looks like this default test has crept in.

Reply:
See 69995fb